src/field_screen_effect.c

```diff
@@ -1223,6 +1224,7 @@ static void Task_OrbEffect(u8 taskId)
         SetGpuReg(REG_OFFSET_DISPCNT, tDispCnt);
         SetGpuReg(REG_OFFSET_BLDCNT, tBldCnt);
         SetGpuReg(REG_OFFSET_BLDALPHA, tBldAlpha);
+        UpdateShadowColor(RGB_BLACK); // force shadows to gray
```

Comment:
There are at least three definitions of what the shadow colour specifically is, and they're all labeled `// force shadows to gray`

Reply:
Settled on adding `RGB_GRAY`, which adds it as `RGB(15, 15, 15)`

Reply:
Doesn't that functionally change the appearance though? Surely there's a reason merrp has different values in different contexts? I don't think making them the same simply because the comment is the same is necessary the right way to deal with this.

Reply:
Hmm, right. I'll make a few checks and report back

Reply:
You're right, this change is a little bit of a regression. I'll remove it and remove the comments.
Before the change
![mGBA_p6jHP004xc](https://github.com/user-attachments/assets/387312a1-d06d-4e19-83dd-44da29aa343a)
After the change:
![mGBA_0JXRi6V0xK](https://github.com/user-attachments/assets/1463f2bf-1511-43c7-a59f-952e600db93a)